app/Http/Controllers/Api/PagesController.php

```diff
@@ -124,4 +124,20 @@ public function showAppDetails($id)
         
         return view('api.pages.appdetails', compact('appDetails'));
     }
+
+    /**
+     * This method delete apps by id
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
+    {
+        $appDetails = AppDetail::where('id', $id)->delete();
+
+        if ($appDetails) {
+            return 200;
```

Comment:
**Code Quality-Nitpick**
@andela-araimi Returning just 200 here is unprofessional something like this would be better there
` return response()->json(['success' => ['message' => 'App Delete', 'code' => 200]], 200)`.